configure.ac

```diff
@@ -0,0 +1,107 @@
+AC_PREREQ(2.61)
+AC_INIT([pjdfstest],[0.1],)
+AC_CONFIG_AUX_DIR([.])
+AM_INIT_AUTOMAKE
+AC_CONFIG_HEADERS([config.h])
+AC_CONFIG_FILES([ \
+    Makefile \
+])
+
+AC_CANONICAL_HOST
+
+AC_PROG_CC
```

Comment:
I haven't run into this issue on my 12.0-CURRENT system (I tried gcc 4.2.1, 4.9, and 6.3.0 from ports). Could you please post the error you receive?